src/software/RigCalibration/main_rigCalibration.cpp

```diff
@@ -90,6 +92,38 @@ std::string myToString(std::size_t i, std::size_t zeroPadding)
   return ss.str();
 }
 
+bool checkRobustEstimator(robust::EROBUST_ESTIMATOR e, double &value)
+{
+  if(e != robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_LORANSAC &&
+     e != robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_ACRANSAC)
+  {
+    POPART_CERR("Only " << robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_ACRANSAC 
+            << " and " << robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_LORANSAC 
+            << " are supported.");
+    return false;
+  }
+  if(value == 0 && 
+     e == robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_ACRANSAC)
+  {
+    // for acransac set it to infinity
+    value = std::numeric_limits<double>::infinity();
+  }
+  // for loransac we need thresholds > 0
+  if(e == robust::EROBUST_ESTIMATOR::ROBUST_ESTIMATOR_LORANSAC)
+  {
+    const double minThreshold = 1e-6;
+    if( value <= minThreshold || value <= minThreshold)
```

Comment:
Just to be sure, it's always better to add a third one:
`if( value <= minThreshold || value <= minThreshold || value <= minThreshold)`